lib/SIL/Parser/ParseSIL.cpp

```diff
@@ -525,15 +595,15 @@ bool SILParser::diagnoseProblems() {
 
     HadError = true;
   }
-  
-  if (!ForwardRefLocalValues.empty()) {
+
+  if (!localValueMap.getForwardRefs().empty()) {
     // FIXME: These are going to come out in nondeterministic order.
-    for (auto &Entry : ForwardRefLocalValues)
+    for (auto &Entry : localValueMap.getForwardRefs())
```

Comment:
This is the comment I was talking about below.

Reply:
I'm planning to submit a patch to start removing forward refs from the test suite after this lands.